homeassistant/components/icloud/config_flow.py

```diff
@@ -101,6 +105,15 @@ def __init__(self):
         if self.api.requires_2sa:
             return await self.async_step_trusted_device()
 
+        try:
+            await self.hass.async_add_executor_job(getattr, self.api, "devices")
+        except (KeyError, PyiCloudNoDevicesException):
```

Comment:
In the real world I am having this : 
<img width="811" alt="Capture d’écran 2020-02-18 à 23 32 01" src="https://user-images.githubusercontent.com/14821482/74784058-5ce8d400-52a7-11ea-85be-c8eaf5d1deaa.png">

But there is also a `NoDeviceException` from the lib here : 

https://github.com/picklepete/pyicloud/blob/9bb6d750662ce24c8febc94807ddbdcdf3cadaa2/pyicloud/services/findmyiphone.py#L66-L67